[2022-11-29 14:28:57,367] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: vaccine_dags.transformData manual__2022-11-29T14:28:53.977517+00:00 [queued]>
[2022-11-29 14:28:57,375] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: vaccine_dags.transformData manual__2022-11-29T14:28:53.977517+00:00 [queued]>
[2022-11-29 14:28:57,376] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 14:28:57,376] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-11-29 14:28:57,376] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 14:28:57,386] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): transformData> on 2022-11-29 14:28:53.977517+00:00
[2022-11-29 14:28:57,390] {standard_task_runner.py:52} INFO - Started process 1680 to run task
[2022-11-29 14:28:57,394] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'vaccine_dags', 'transformData', 'manual__2022-11-29T14:28:53.977517+00:00', '--job-id', '1234', '--raw', '--subdir', 'DAGS_FOLDER/vaccine.py', '--cfg-path', '/tmp/tmpc8pmpsjw', '--error-file', '/tmp/tmp0m_xwazr']
[2022-11-29 14:28:57,394] {standard_task_runner.py:77} INFO - Job 1234: Subtask transformData
[2022-11-29 14:28:57,447] {logging_mixin.py:109} INFO - Running <TaskInstance: vaccine_dags.transformData manual__2022-11-29T14:28:53.977517+00:00 [running]> on host acb48b0a6e0d
[2022-11-29 14:28:57,489] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vaccine_dags
AIRFLOW_CTX_TASK_ID=transformData
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T14:28:53.977517+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T14:28:53.977517+00:00
[2022-11-29 14:28:57,522] {python.py:175} INFO - Done. Returned value was: None
[2022-11-29 14:28:57,532] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=vaccine_dags, task_id=transformData, execution_date=20221129T142853, start_date=20221129T142857, end_date=20221129T142857
[2022-11-29 14:28:57,567] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-29 14:28:57,602] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
